.buildbot.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+#
+# Build script for continuous integration.
+
+./x.py clean  # We don't clone afresh to save time and bandwidth.
+
+# Note that the gdb must be Python enabled.
+PATH=/opt/gdb-8.2/bin:${PATH} RUST_BACKTRACE=1 YK_DEBUG_SECTIONS=1 \
```

Comment:
I added the `YK_DEBUG_SECTIONS` env here so that we test the compiler with the `.yk_mir_cfg` section enabled. Also switched to full stage 2 tests.